### PR TITLE
redenomination updates for runtime main, staking

### PR DESCRIPTION
**Pull request by rajesh-nodle**
redenomination updates for  main & staking runtime ..., updated to 1 nodle = 10 ** 11.

### CODE REVIEW

File: pallets/wnodl/Cargo.toml [hunk 1]
```diff
@@ -21,11 +21,11 @@ sp-std = { git = "https://github.com/paritytech/substrate", default-features = f
 sp-core = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.17" }
 
 support = { path = "../../support" }
+pallet-reserve = { path = "../../pallets/reserve", optional = true, default-features = false }
+pallet-balances = { git = "https://github.com/paritytech/substrate", optional = true, default-features = false , branch = "polkadot-v0.9.17" }
 
 [dev-dependencies]
 sp-io = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.17" }
-pallet-balances = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.17" }
-pallet-reserve = { path = "../../pallets/reserve", default-features = false }
```
Comment (aliXsed) [inline pallets/wnodl/Cargo.toml:29]: Not a problem for this PR buy just wonder why this change was needed?

Comment (rajesh-nodle) [reply pallets/wnodl/Cargo.toml:29]: Causing build failure with runtime 'main'

Comment (aliXsed) [reply pallets/wnodl/Cargo.toml:29]: Then the question is how it was building before!? Do you know what might have changed?

Comment (rajesh-nodle) [reply pallets/wnodl/Cargo.toml:29]: Wondering, how it works earlier .

**Comment by aliXsed** [comment]
Closing this as the plan for our staking pallet would revisit everything including using a parity pallet instead of our own.
